inference-engine/thirdparty/CMakeLists.txt

```diff
@@ -40,6 +40,7 @@ if (ENABLE_CLDNN)
         set(CLDNN__ARCHITECTURE_TARGET "Linux64" CACHE STRING "" FORCE)
     endif()
     set(CLDNN_THREADING "${THREADING}" CACHE STRING "" FORCE)
+    set(GPU_DEBUG_CONFIG OFF CACHE BOOL "Enable debug config feature")
```

Comment:
Maybe this option should be OFF by default for Release build and ON for Debug? It will allow us to check compilation of debug code in CI.

Reply:
Good idea, can I follow-up in the next PR? Hope to merge this infra first.

Reply:
Sure